config/sync/filter.format.basic_html.yml

```diff
@@ -92,12 +95,37 @@ filters:
     weight: -38
     settings:
       replace_empty: '0'
+  media_embed:
+    id: media_embed
+    provider: media
+    status: true
+    weight: -48
+    settings:
+      default_view_mode: embed
```

Comment:
@johangant I've created a PR that adds in additional view modes that editors will be able to use in CKEditor - https://github.com/dof-dss/nidirect-drupal/pull/315